ultravox/training/evaluation.py

```diff
@@ -44,14 +45,36 @@ def dataset_infer(
     return ddp_utils.all_gather_list(eval_samples)
 
 
-def get_metric_name(ds_name: str, metric: str) -> str:
-    if ds_name == "boolq_in" and metric == "asr":
-        return "boolq__wer"
-    if ds_name == "boolq" and metric == "boolq":
-        return "boolq__correctness"
-    if metric == "instruct":
-        return f"{ds_name}__instruct_follow"
-    return f"{ds_name}__{metric}"
+@dataclasses.dataclass
+class EvalScenario:
+    name: str
+    dataset: str
+    metric: str
+    include_audio: bool = True
+    include_context: bool = True
+    new_tokens: Optional[int] = None
+
+
+EVAL_SCENARIOS = [
+    EvalScenario("boolq__wer", "boolq_in", "asr"),
+    EvalScenario("anyinstruct__instruct_follow", "anyinstruct", "instruct"),
+    EvalScenario(
+        "boolq_binary", "boolq_passage", "exact_match_last_word", new_tokens=128
```

Comment:
What's the thinking with one vs two underscores? Should this be `boolq__binary`?

Reply:
You're right, I missed that one.

> What's the thinking?

I was mainly trying to match the strings I had before.

Reply:
Also, `_` in my mind separates compound words, whereas `__` separates different concepts, so here it was: `{dataset}__{metric}[__text_only]`.